Package.swift

```diff
@@ -32,7 +32,7 @@ let package = Package(
     ),
   ],
   dependencies: [
-    .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0"),
+    .package(url: "https://github.com/swiftlang/swift-docc-plugin", from: "1.0.0"),
```

Comment:
Anyone know if this causes problems if some project depending on us ends up with two things depending on docc via different urls?

Reply:
I'm hoping the Apple folks can chime in on if this would be a problem or not.

Reply:
Good question. I haven't tried this out yet.

Reply:
We should update it.

If different URLs are a problem, that's true regardless of whether we update, since other people will.